### PR TITLE
Add a pod YAML file for the submit-queue.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

**Comment by brendandburns** [comment]
@ixdy @zmerlynn can I get a quick review?

File: submit-queue/pod.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+metadata:
+  name: submit-queue
+  namespace: default
+spec:
+  containers:
+  - image: gcr.io/google_containers/submit-queue:0.1
+    command:
+      - bash
+      - -c
+      - "/submit-queue --user-whitelist=/whitelist.txt --jenkins-host=http://jenkins-master:8079 --token=$(cat /etc/secret-volume/token) --alsologtostderr"
```
Comment (ixdy) [inline submit-queue/pod.yaml]: where does this jenkins-host address come from?

Comment (brendandburns) [inline submit-queue/pod.yaml]: That's the name of the VM that the jenkins instance runs on.  The cluster runs in the same GCE project, so it just uses the built in project DNS.

Comment (ixdy) [inline submit-queue/pod.yaml]: OK, but I think Jenkins uses port 8080, not 8079?

Comment (ixdy) [inline submit-queue/pod.yaml]: also, won't passing the token in as a cmdline argument expose it via ps?

Comment (brendandburns) [inline submit-queue/pod.yaml]: Switched to 8080 (I swear it was 8080 before...)

wrt to the command line, that's true, and we should prob. fix eventually, but for now, if you can log onto the machine, you can talk to the kubernetes API server and read the secret anyway.

**Comment by brendandburns** [comment]
@ixdy comments addressed, please take another look.

Thanks!
--brendan

**Pull request merged**